src/reducers/ast.js

```diff
@@ -151,7 +151,10 @@ function update(
 // https://github.com/devtools-html/debugger.html/blob/master/src/reducers/sources.js#L179-L185
 type OuterState = { ast: Record<ASTState> };
 
-export function getSymbols(state: OuterState, source: Source): ?Symbols {
+export function getSymbols(
+  state: OuterState,
+  source: Source
+): ?SymbolDeclarations {
```

Comment:
It's important that this is `Symbols` and not `SymbolDeclarations` look at the definition above

Reply:
hey @jasonLaster were you referring to only this case or the other ones as well. I may have read this wrong initially.. I just realized you may have meant only this case and not the others .. oops. Please clarify this for me I can make a quick fix